apps/server/src/modules/files-storage/controller/files-storage.controller.ts

```diff
@@ -74,6 +85,7 @@ export class FilesStorageController {
 	}
 
 	@Patch('/rename/:fileRecordId/')
+	@UseInterceptors(RequestLoggingInterceptor)
```

Comment:
Why do we apply the logging interceptor only to  a subset of the controller actions?

Reply:
We're not allowed to log all requests, so the interceptor is specific to API calls